pages/FAQ.md

```diff
@@ -0,0 +1,7 @@
+# FAQ Documentation
+
+Since the inception of v4, we have been compiling a list of relevant questions and answers that are asked by MMs/Trading Firms on Telegram. We turned this list into a live _v4 Telegram Questions_ document, which should be able to answer many of your more nuanced questions.
```

Comment:
Replace "v4" with "dYdX Chain" , change "relevant questions and answers that are asked by MMs/Trading Firms on Telegram," to "questions and answers from users related to technical support" - take "v4 Telegram" out of doc title and call it "User Support Questions" or something